bbs/register/action.php

```diff
@@ -1,6 +1,6 @@
 <?php
-include("../lib/mainfunc.php");
-include("../../assets/api/captcha.php");
+require_once "../lib/mainfunc.php";
+require_once "../../assets/api/captcha.php";
```

Comment:
Can you explain more detail about whether you use `require_once` or just `require`?